apps/web/components/Navbar/Bottom.tsx

```diff
@@ -0,0 +1,18 @@
+import { Flex, Text } from '@chakra-ui/react';
+import React from 'react';
```

Comment:
Why. React 17 comes has already stated this. You dont need do import react everywhere. It will automatically be there

Reply:
i uses "rfce" command to create components so it comes along with command

Reply:
Hihi. Update karo karoo 😆